docs: setup structure, installation

royalpinto007 opened the pull request:
I made some changes to the installation files to enhance the formatting for a cleaner look. Additionally, I replaced the logo with a transparent version to eliminate the background and addressed some issues identified in Korel's PR- https://github.com/dentproject/docs.dent.dev/pull/6.

taraschornyiplv commented on Installation/LocalMachine.md:
```diff
@@ -0,0 +1,76 @@
+
+<div style="border-top: 1px solid black;"></div>
+
+For more information, feel free to contact the author [Royal Simpson Pinto](mailto:royalpinto007@gmail.com).
```
i'd rather put reference to dent.dev or github

royalpinto007 replied:
fixed- https://github.com/dentproject/docs.dent.dev/pull/7/commits/d74549edf4ffd2b16a68b4e2f2331a431684667e

taraschornyiplv commented on Installation/VirtualMachine.md:
```diff
@@ -0,0 +1,89 @@
+
+### You have now successfully set up DENT in GNS3 with a virtual machine
+
+For more information, feel free to contact the author [Korel Ucpinar](mailto:Korelucpinar@gmail.com).
```
same here

taraschornyiplv commented on index.md:
```diff
@@ -1,35 +1,32 @@
 
-- uses a gem-based approach, i.e. uses a `Gemfile` and loads the `just-the-docs` gem
-- uses the [GitHub Pages / Actions workflow] to build and publish the site on GitHub Pages
+With the second release “Beeblebrox” DENT now includes
```
can we also add info about 3rd release?

royalpinto007 replied:
Sure, will do that!

royalpinto007 commented:
Considering the revised changes in https://github.com/dentproject/docs.dent.dev/pull/6, from https://github.com/dentproject/docs.dent.dev/pull/6/commits/24ed65d0a8178d3851b9ac4559233af97ba583e8 to https://github.com/dentproject/docs.dent.dev/pull/6/commits/c868b1dac39f766beec771775854e9b014813952,  I'm closing this PR. I'll be working on a separate PR to incorporate all my changes, focusing specifically on **LocalMachine.md** and **index.md**. These are the files I primarily worked on along with addressing issues in other files, along with some of the images that Korel missed from my [Medium article](https://medium.com/@royalpinto007/how-to-install-dent-nos-on-gns3-e6bac65bc056).

Please ensure that the files I've worked on are properly addressed so that I get credit for my contributions. Alternatively, if feasible, you could cherry-pick the changes from this PR. However, I believe a separate PR from me would streamline the process for you. @taraschornyiplv
